src/sentry/logging/handlers.py

```diff
@@ -48,13 +48,23 @@ def __call__(self, logger, name, event_dict):
 
 class StructLogHandler(logging.StreamHandler):
     def emit(self, record, logger=get_logger()):
+        # This may seem a little gross, but when logging with the RootLogger,
+        # anything passed into 'extra' gets turned into a real kwarg
+        throwaways = [
+            'threadName', 'thread', 'created', 'process', 'processName', 'args',
+            'module', 'filename', 'levelno', 'exc_text', 'msg', 'pathname', 'lineno',
+            'funcName', 'relativeCreated', 'levelname', 'msecs',
+        ]
+
         kwargs = {
+            (k if k not in throwaways else 'popme'): v
```

Comment:
This is `O(n*m)`. Declare `throwaways` as a `frozenset` at module level since it's a constant. If it's a set, it'll go down to `O(n)` instead.